IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DoorTypeTurnstile/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_DoorTypeTurnstile" UniqueId="c5233710-61f4-8eb7-185b-60b03c52ca80" ApplicableType="IfcDoor/TURNSTILE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsBidirectional_3PYJ6a4gXACx94JYuDo4no" />
+		<DocProperty xsi:nil="true" href="GateID_1vaewDvSn2zgrPRZgB_Hg9" />
```

Comment:
im not sure about adding additional identifiers in a PSet would a strategy using Name or Tag attribute be sufficient?

Reply:
Tag could be sufficient.

Reply:
Additionally, Consider reuse of Reference property on Pset_DoorCommon